spec/javascripts/visibility_component_spec.js

```diff
@@ -169,6 +169,25 @@ describe("VisibilityComponent", function() {
         expect(target.requireEmbargo).toHaveBeenCalledWith("authenticated", futureDate);
       });
     });
+    describe("with required past release date, dont restrict visibility", function() {
+      beforeEach(function() {
+        spyOn(target, 'disableEmbargoAndLease');
+      });
+      it("require visibility", function() {
```

Comment:
Tiny thing, but this `it` description is incorrect. Looks like it should say "require immediate release" or "disable embargo and lease"